test/cli/simple-package/test.out

```diff
@@ -5,16 +5,15 @@ foo/foo.rb:13: Unable to resolve constant `BardClass` https://srb.help/5002
     foo/foo.rb:13: Replace with `Project::Bar::BarClass`
     13 |    Project::Bar::BardClass
             ^^^^^^^^^^^^^^^^^^^^^^^
-    bar/__package.rb: `Project::Bar::BarClass` defined here
+    bar/bar.rb:6: `Project::Bar::BarClass` defined here
+     6 |  class BarClass
+          ^^^^^^^^^^^^^^
 
-foo/foo.rb:14: Package `Project::Bar` does not export `Project::Bar::UnexportedClass` https://srb.help/5002
+foo/foo.rb:14: Package `Project::Bar` does not export `Project::Bar::UnexportedClass` https://srb.help/3717
     14 |    Project::Bar::UnexportedClass
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   Autocorrect: Use `-a` to autocorrect
     bar/__package.rb:9: Insert `export Project::Bar::UnexportedClass`
      9 |  export Project::Bar::BarMethods
                                          ^
-    bar/bar.rb:14: Defined here
-    14 |  class UnexportedClass; end
-          ^^^^^^^^^^^^^^^^^^^^^
```

Comment:
is it worth preserving this error section in the new error?

Reply:
I think that would be an easy fix 👍